src/_incydr_sdk/actors/client.py

```diff
@@ -177,6 +181,43 @@ def get_family_by_member_name(self, name: str) -> ActorFamily:
             if err.response.status_code == 404:
                 raise ActorNotFoundError(name)
 
+    def update_actor(
```

Comment:
I think we should consider defaulting the `notes` and `date` params here to `None` and raise an exception if none of them are passed in (similar to the CLI)

Reply:
It could also be nice to add a method signature overload in addition to this update method that takes an `Actor` object so that people could
do something like
`actor = client.get_actor(id)`
`actor.start_date = "01-01-1111"
`client.update(actor)`

Reply:
It looks like the cases client just has one method that accepts the case. But I think the overload is a nice idea (if a bit awkward to implement in python), so it is added now.